### PR TITLE
Fix 755 browse grammar

### PR DESCRIPTION
**Pull request by irina060981**
For the issue https://github.com/alpheios-project/components/issues/755
- Fix small bug with back button
- Add show/hide grammar titles

### CODE REVIEW

File: src/vue/components/grammar.vue [hunk 1]
```diff
@@ -50,6 +57,7 @@ import { Grammars } from 'alpheios-res-client'
 import Vue from '@vue-runtime'
 
 import BackIcon from '@/images/inline-icons/back.svg'
+import DefinitionsIcon from '@/images/inline-icons/definitions.svg'
```
Comment (balmas) [inline src/vue/components/grammar.vue]: shouldn't this be the grammar icon instead?

Comment (irina060981) [reply src/vue/components/grammar.vue]: oh, I don't know why I mixed them up!

Comment (irina060981) [reply src/vue/components/grammar.vue]: Done 
image

**Comment by irina060981** [comment]
I think as there was only one small fix after review - I could go ahead and merge it!

**Pull request merged**